addons/website_sale_autocomplete/static/src/interactions/address_form.js

```diff
@@ -9,7 +9,7 @@ class AddressForm extends Interaction {
     static selector = ".oe_cart .checkout_autoformat";
     static selectorHas = "input[name='street'][data-autocomplete-enabled='1']";
     dynamicContent = {
-        "input[name='street']": { "t-on-input": (ev) => this.debounced(this.onInputStreet(ev.currentTarget), 200) },
+        "input[name='street']": { "t-on-input": this.debounced(this.onInputStreet, 200) },
```

Comment:
We'll need a "current target" extraction tool:
```js
(ev) => this.targetted(this.debounced(this.onInputStreet, 200))
[...]
async onInputStreet(ev, currentTarget) {
```
or always include it in throttled & debounced calls ?